agent/exploits/cve_2024_6386.py

```diff
@@ -0,0 +1,35 @@
+"""Agent Asteroid implementation for CVE-2024-6386"""
+
+import re
+
+from agent import definitions
+from agent import exploits_registry
+from agent.exploits import webexploit
+
+VULNERABLE_VERSIONS_REGEX = r"(?m)^Stable tag:\s+(0|[1-3](?:\.\d+){0,2}|4(?:\.(?:[0-5](?:\.\d+)?|6(?:\.(?:[0-9]|1[0-2]))?)?)?)$"
```

Comment:
Oh my god. This is unreadable and should be burnt with fire :fire: .

Split each version with a separate regex :fire_extinguisher: , prioritize readability over writing smart regexp please.

Reply:
Done.